tests/sys/fs/fusefs/ioctl.cc

```diff
@@ -0,0 +1,190 @@
+/*-
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright (c) 2019 The FreeBSD Foundation
+ *
+ * Portions of this software was developed by BFF Storage Systems, LLC under
+ * sponsorship from the FreeBSD Foundation.
```

Comment:
You should update this copy/pasted copyright section.

Reply:
(That said, if there is a reason you do not want to be the copyright holder I can discuss that separately.)